extensions/npm/package.json

```diff
@@ -88,6 +88,10 @@
       {
         "command": "npm.runSelectedScript",
         "title": "%command.runSelectedScript%"
+      },
+      {
+        "command": "npm.runScriptsFromFolder",
+        "title": "%command.runScriptsFromFolder%"
```

Comment:
a nit but `runScriptsFromFolder` should not be plural, the command will run only a single script.